pyccl/base/repr_.py

```diff
@@ -206,7 +206,7 @@ def build_string_Pk2D(self, na=6, nk=6, decimals=2):
             +===============+=============================================+
     """
     if not self.has_psp:
-        return "pyccl.Pk2D(empty=True)"
+        return "pyccl.Pk2D(empty)"
```

Comment:
There is an `empty=True` in the `Tk3D` representation as well.

Reply:
Yep, fixed.